pkg/tsdb/cloudwatch/log_query.go

```diff
@@ -112,7 +129,9 @@ func groupResults(results *data.Frame, groupingFieldNames []string) ([]*data.Fra
 func generateGroupKey(fields []*data.Field, row int) string {
 	groupKey := ""
 	for _, field := range fields {
-		groupKey += *field.At(row).(*string)
+		if strField, ok:= field.At(row).(*string); ok {
+			groupKey += *strField
```

Comment:
Need to check that strField is not nil I believe (unless you are sure), or panic will happen. An interface can be of type (*string) can still have a nil value.

Reply:
Example: https://play.golang.org/p/wr5JdZpEmc1

Reply:
Good point @kylebrandt.